deploy/ArbitrumDeploy.js

```diff
@@ -55,9 +55,42 @@ module.exports = async (hre) => {
         from: deployer,
         log: true,
         contract: "ChainlinkOracleWrapper",
-        args: [KovanEurUsdOracle.address],
+        args: [KovanEurUsdOracle.address, deployer],
     })
 
+    // deploy SMA PriceObserver
+    const priceObserver = await deploy("EthUsdPriceObserver", {
+        from: deployer,
+        log: true,
+        contract: "PriceObserver",
+    })
+
+    // deploy SMA Oracle
+    const smaOracleWrapper = await deploy("EthUsdPriceSMAOracle", {
+        from: deployer,
+        log: true,
+        contract: "SMAOracle",
+        args: [
+            RinkebyEthUsdOracle.address,
+            8,
+            priceObserver.address,
+            12,
+            3600,
```

Comment:
Inline comments next to these parameters would be greatly beneficial to the readability of this script.